Makefile

```diff
@@ -2,6 +2,7 @@ SHELL := $(shell which bash) # Use bash syntax to be consistent
 
 OS_NAME := $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH_NAME_RAW := $(shell uname -m)
+# TODO: remove this in upcomming next versions
```

Comment:
We won't remove this for a very, very long time; if ever. Instead, can we add a comment that mentions that we publish to that repo for backwards compatibility?

Reply:
Sure